package.json

```diff
@@ -199,5 +195,14 @@
       "prettier --write",
       "git add"
     ]
+  },
+  "jest": {
+    "rootDir": "src",
+    "modulePaths": [
+      "<rootDir>/"
```

Comment:
not `<rootDir>/src`? 

`https://github.com/steemit/condenser/pull/2235/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R200`

Reply:
yup; changed the rootDir above in e60d6bf